estuary-cdk/estuary_cdk/capture/request.py

```diff
@@ -28,15 +28,28 @@ class Discover(GenericModel, Generic[EndpointConfig]):
 class ValidateBinding(GenericModel, Generic[ResourceConfig]):
     collection: CollectionSpec
     resourceConfig: ResourceConfig
+    backfill: NonNegativeInt = 0
 
 
-class Validate(GenericModel, Generic[EndpointConfig, ResourceConfig]):
+class ValidateBase(GenericModel, Generic[ResourceConfig]):
     name: str
     connectorType: ConnectorType
-    config: EndpointConfig
     bindings: list[ValidateBinding[ResourceConfig]] = []
 
 
+class ValidateLastCaptureEndpointConfig(BaseModel):
+    config: dict[str, Any]
+
+
+class ValidateLastCapture(ValidateBase, Generic[EndpointConfig, ResourceConfig]):
+    config: ValidateLastCaptureEndpointConfig
+
+
+class Validate(ValidateBase, Generic[EndpointConfig, ResourceConfig]):
+    config: EndpointConfig
+    lastCapture: ValidateLastCapture[EndpointConfig, ResourceConfig] | None = None
+
+
```

Comment:
Updated Validate models to include `lastCapture` spec. Currently uses a work around due to issues serializing `lastCapture` into the `EndpointConfig` model.